<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioPerfiles</class>
 <widget class="QDialog" name="FormularioPerfiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>106</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>A&amp;vanzada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>189</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Configurando perfil</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>431</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tiempo por hora</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="tPorHora"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nombre"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hr entrada</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="hrEntrada"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hr salida</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="hrSalida"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string># horas</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="nHoras"/>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tiempo receso</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="tReceso"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="confirmar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>166</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
